From f5e5519a04da0cf198c77756adb339e3d202cd78 Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Mon, 2 Mar 2020 13:24:47 +0100
Subject: [PATCH] Fix openssl command for generating EC key

The current command generates PEM file with both EC parameters and the
key and it results in an invalid CSR.
---
 cesnet-tcs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cesnet-tcs b/cesnet-tcs
index 35f093a..9ff1457 100755
--- a/cesnet-tcs
+++ b/cesnet-tcs
@@ -193,7 +193,7 @@
 
 	case "$key_alg" in
 		rsa:*) openssl genrsa -out "$outfile" ${key_alg#*:};;
-		*) openssl ecparam -name "$key_alg" -genkey -param_enc explicit -out "$outfile";;
+		*) openssl ecparam -genkey -name "$key_alg" -noout -out "$outfile";;
 	esac
 
 	umask "$orig_umask"
